modules/core/src/test/java/org/apache/ignite/internal/processors/cache/ActiveOnStartPropertyTest.java

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright 2019 GridGain Systems, Inc. and Contributors.
+ *
+ * Licensed under the GridGain Community Edition License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.processors.cache;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.apache.ignite.configuration.CacheConfiguration;
+import org.apache.ignite.configuration.DataRegionConfiguration;
+import org.apache.ignite.configuration.DataStorageConfiguration;
+import org.apache.ignite.configuration.IgniteConfiguration;
+import org.apache.ignite.internal.util.typedef.internal.CU;
+import org.apache.ignite.testframework.ListeningTestLogger;
+import org.apache.ignite.testframework.LogListener;
+import org.apache.ignite.testframework.junits.common.GridCommonAbstractTest;
+import org.junit.Test;
+
+import static org.apache.ignite.cluster.ClusterState.ACTIVE;
+import static org.apache.ignite.cluster.ClusterState.INACTIVE;
+
+/**
+ * Checks that {@link IgniteConfiguration#isActiveOnStart()} works correctly after deprecation and introduction {@link
+ * IgniteConfiguration#getClusterStateOnStart()}.
+ */
+public class ActiveOnStartPropertyTest extends GridCommonAbstractTest {
+    /** Active on start flag. */
+    private boolean activeOnStart;
+
+    /** Persistence enabled flag. */
+    private boolean persistenceEnabled;
+
+    /** */
+    private Map<String, LogListener> logListeners = new HashMap<>();
+
+    /** {@inheritDoc} */
+    @Override protected IgniteConfiguration getConfiguration(String igniteInstanceName) throws Exception {
+        ListeningTestLogger testLog = new ListeningTestLogger(false, log);
+
+        LogListener lsnr = LogListener.matches(
+            "Property activeOnStart deprecated. Use clusterStateOnStart instead."
```

Comment:
use static final fields for comparison.

Reply:
I disagree with it. Usage strings make tests more readable.